src/components/TouchBar/FocusMenu/FocusButton.jsx

```diff
@@ -27,12 +27,17 @@ class FocusButton extends Component {
   }
 
   select() {
-    this.isActive ? this.props.onRefocus(this.props.descriptionFlyTo) : this.props.onChangeOrigin(this.props.identifier);

```

Comment:
storyTouch version might need this?

Reply:
Yes, but I think the functionality is intact. My interpretation of the old code was that "if the button is active and is clicked again, then only apply the fly-to-function. If not active, change the origin (anchor/focus) to id of this button and trigger the fly-to for the new origin". I changed the logic a bit so the select function does not really differentiate if the selected button is already in focus or not, but basically says whichever button is selected, set that node as origin/anchor/focus and fly to it. Sure, it may be a little unnecessary to change the origin if I reclick the same button, but it is not a very costly function, or super common use case that will hinder the performance imo. The way OnRefocus was passed down through props seemd to add unnecessary complexity.